app/ui-account/client/accountPreferences.js

```diff
@@ -346,4 +353,44 @@ Template.accountPreferences.events({
 
 		selectEl.remove(optionIndex);
 	},
+	'click .alexa-token-generate'(e, instance) {
+		e.preventDefault();
+
+		const serverName = instance.find('#alexaServerName').value.trim();
+
+		if (!serverName) {
+			return toastr.error(t('Register_Alexa_Servername_Error'));
+		}
+
+		const tokenName = `alexa-${ serverName }`;
+
+		Meteor.call('personalAccessTokens:generateToken', { tokenName }, (error, token) => {
```

Comment:
You could call this on the server-side as well rather than call if from the client, wait for the server's response then pass it again to the server.

Reply:
done